model/DBHelper.cs

```diff
@@ -39,7 +41,16 @@ public class DBHelper {
 			}
 		}
 
-		public static void DropDb(string conn) {
+	    public static void RunSqlWithNoExec(string conn, string sql) {
```

Comment:
I am not sure this should be on a production class, you are only using it in tests at this point.  I would move it to the testing project.